bin/filter-preview-output.sh

```diff
@@ -6,4 +6,6 @@ grep -v -e '^\.\.\.0 corrections$' |\
   grep -v -e '\.\.\. (skipping)' |\
   grep -v -e '^----- DNS Provider: ' |\
   grep -v -e '^----- Registrar: ' |\
-  grep -v -e '^----- Getting nameservers from:'
+  grep -v -e '^----- Getting nameservers from:' | \
```

Comment:
The `grep -v` statements can be removed.  Those are only output if the `--full` flag is used, which it isn't.